js/src/dapps/registry/events/reducers.js

```diff
@@ -0,0 +1,24 @@
+const initialState = [];
+
+const sortEvents = (a, b) => {
+  const d = a.block.minus(b.block).toFixed(0);
+  if (d === 0) return a.index.minus(b.index).toFixed(0);
+  return d;
```

Comment:
I disagree with `if (d)`. It is unintuitive, as you have to know how JS converts number to boolean.